openedx_lti_tool_plugin/templates/openedx_lti_tool_plugin/courseware.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+  </head>
+
+  <body>
+    <iframe
+      id='unit-iframe'
+      src='{{ xblock_url }}'
+      allow='microphone *; camera *; midi *; geolocation *; encrypted-media *'
+      allowfullscreen=''
+      scrolling='no'>
+    </iframe>
+    <style>
+      #unit-iframe {
+          width: 100%;
+          border: none;
+          display: block;
+      }
+    </style>
+    <script>
+      // Detect message events from the XBlock iframe.
```

Comment:
Why is this required? @kuipumu

Reply:
This receives the Window.postMessage() sent on the render_xblock template view lms/templates/courseware/courseware-chromeless.html, with this we can receive the plugin.resize message to get the height and width reported by the iframe, with this we can have a responsive height by making the parent window change the iframe size dynamically according to the iframe reported size.